tests/pom.xml

```diff
@@ -875,6 +902,79 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
                     </wait>
                   </run>
                 </image>
+                <!-- ##### Command Router service (may also act as Device Connection service via 'enable-device-connection-endpoint') ##### -->
+                <image>
+                  <name>${docker.repository}/hono-service-command-router-test</name>
+                  <alias>command-router</alias>
+                  <build>
+                    <imagePullPolicy>IfNotPresent</imagePullPolicy>
```

Comment:
we should also skip building the image if the command router is disabled ...

Reply:
Done